cli/pack/pack.go

```diff
@@ -169,7 +170,7 @@ func Run(ctx *context.Ctx) error {
 	return nil
 }
 
-func FillCtx(ctx *context.Ctx) error {
```

Comment:
Good, but `cli/pack` shouldn't know anything about `flags *pflag.FlagSet` (it's used by `cli/commands` module). Let's move these checks to `cli/commands/pack`.